tests/client_test.go

```diff
@@ -412,6 +412,32 @@ func TestStats(t *testing.T) {
 		t.Errorf("Error getting stats: %v", err)
 	}
 
+	// Status info
+	if stats.Status.Version == "" {
+		t.Error("Missing version string")
+	}
+	if stats.Status.Build == "" {
+		t.Error("Missing build string")
+	}
+	if stats.Status.Address != "127.0.0.1" {
```

Comment:
Better to check for non empty string to not make any assumptions about NGINX IP address.